packages/web/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1378,6 +1392,33 @@ class AudiusAPIClient {
     return response ? response.data : null
   }
 
+  async getReaction({ reactedToIds }: GetReactionArgs) {
+    const params = {
+      reacted_to_ids: reactedToIds
+    }
+    const response: Nullable<APIResponse<
+      GetReactionResponse
+    >> = await this._getResponse(
+      FULL_ENDPOINT_MAP.getReaction,
+      params,
+      false,
+      PathType.VersionFullPath,
+      {},
+      true
+    ) // Perform without retries, using 'split' approach for multiple query params
+
+    if (!response || !response.data.length) return null
+
+    const adapted = response.data.map(item => ({
+      reactionValue: parseInt(item.reaction_value),
+      reactionType: item.reaction_type,
+      senderUserId: decodeHashId(item.sender_user_id),
+      reactedTo: item.reacted_to
+    }))[0]
```

Comment:
wondering why we pass an array of ids if we always just read the first one?

Reply:
great q - this endpoint is sort of broken rn, in that it accepts multiple tx signatures but only returns the first one. I have a linear tracking fixing this